app/src/main/java/org/thoughtcrime/securesms/home/search/GlobalSearchInputLayout.kt

```diff
@@ -16,42 +16,37 @@ import android.widget.TextView
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import network.loki.messenger.databinding.ViewGlobalSearchInputBinding
+import org.thoughtcrime.securesms.util.SimpleTextWatcher
+import org.thoughtcrime.securesms.util.addTextChangedListener
 
 class GlobalSearchInputLayout @JvmOverloads constructor(
```

Comment:
This search input field is very narrow. The designs have it larger. 
The 'x' button is also hard to tap on and probably has a click area too small

Reply:
fixed

<img width="360" alt="Screen Shot 2024-06-15 at 1 29 06 am" src="https://github.com/oxen-io/session-android/assets/9282178/552d0fb7-04be-4d8d-920b-78ad8ff49b2a">